spond/spond.py

```diff
@@ -378,3 +378,22 @@ async def update_event(self, uid, updates: dict):
         ) as r:
             self.events_update = await r.json()
             return self.events
+
+    @SpondBase.require_authentication
+    async def get_export(self, uid: str) -> bytes:
```

Comment:
Think this is better named something more specific like `get_event_attendance_xlsx()` @Olen?

Reply:
yep more explicit i changed it